src/common/evaluation/QA/eval_depthmap_models/src/utils.py

```diff
@@ -295,6 +299,35 @@ def draw_age_scatterplot(df_: pd.DataFrame, csv_out_fpath: str):
     plt.close()
 
 
+def draw_stunting_diagnosis(df: pd.DataFrame, csv_out_fpath: str):
+    predicted_stunting = []
+    actual_stunting = []
+    not_processedData = []
+    for index, row in df.iterrows():
+        sex = 'M' if row[COLUMN_NAME_SEX] == SEX_DICT['male'] else 'F'
+
+        if row['GT'] > 45 and row['GT'] <= 120 and row['predicted'] > 45 and row['predicted'] <= 120 and row[COLUMN_NAME_AGE] <= 1856.0:
```

Comment:
To make this line more readible:
```suggestion
        if 45< row['GT'] <= 120 and 45 < row['predicted'] <= 120 and row[COLUMN_NAME_AGE] <= 1856.0:
```